### DISCUSSION TITLE
only ten row dysplay in dataset when I "run SQL"

### DISCUSSION THREAD

**Discussion by PhilippeDo** [cuebook/CueObserve#173] [General]
Hi, 

In  dataset panel when I run SQL, I have always 10 rows dysplayed, despite that I have more than 10 rows in my database. It seems that the lift on the right does'nt works.

![image](https://user-images.githubusercontent.com/22300126/136768615-97ad6809-86aa-498c-951e-a8ab12712603.png)

**Comment by sachinkbansal** [comment]
@PhilippeDo This is by design. We limit display rows to 10 in the dataset screen to avoid slowing down the UI. This doesn't impact the actual rows in your dataset.

**Reply by PhilippeDo** [reply]
OK good to know 
thanks you.
